message_app_client/src/routes/MessagesPage.js

```diff
@@ -12,16 +14,19 @@ const conversations = {conversations: [
 ]}
 
 function MessagesPage() {
-  const { currentUser } = useContext(SessionDataContext);
+  const [ conversationID, setConversationID ] = useState();
+  const { currentUserID } = useContext(SessionDataContext);
 
   return (
     <div className='container'>
+      <ConversationDataContext.Provider value={{ conversationID, setConversationID }}>
         <ConversationPanel {...conversations}/>
         <ConversationDetailPanel
           conversationId={1}
-          username={currentUser}
+          username={"user1"}
           otherParticipantName={"user2"}
```

Comment:
So it is hard coded for testing?

Reply:
Yes, it is hardcoded for testing. That value came from @vuongtranluc last PR. For now, I keep that testing value so that the application will not raise any errors. However, in the future, I will have to make a few modifications to other components in the future, for we changed the state from `currentUser` to `currentUserID`

Reply:
Make a ticket for it so that we do not forget.